chukwuemekachm/M-Tracker-React-Client
# #160296219 Completes view single request feature

#### What does this PR do?
- This PR completes view single request feature
#### Description of Task to be completed?
- Add ViewSingleRequest.jsx
- Add ViewSingleRequestPage.jsx
- Test components
- Update existing tests
- Update existing snapshots
#### How should this be manually tested?
- Clone the repo
- Startup command line/Terminal
- Navigate to the project's directory
- Run `npm install`
- Run `npm start` or `npm run dev`
- Your default browser will be opened if you used `npm run dev`
- Login or signup(navigate to `/login` or `/signup`)
- Click on any request on your dashboard or create a new request and click it
- Observe the single request display page
#### Production URL
- [heroku](https://m-tracker-react.herokuapp.com)
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#160296219](https://www.pivotaltracker.com/story/show/160296219)
#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-10-02 at 12 52 00 am" src="https://user-images.githubusercontent.com/33798252/46321973-6c964880-c5de-11e8-8410-24c7c74bd61a.png">


<img width="1440" alt="screen shot 2018-10-02 at 12 52 14 am" src="https://user-images.githubusercontent.com/33798252/46321975-6c964880-c5de-11e8-82b7-8fd8848664c1.png">

#### Questions:
- None
